.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+* [Link to the GitHub issue it solves (if any)]
+* Make sure the following build targets pass:
```

Comment:
For me, this looks more like a guidelines to raise than a PR template.

Reply:
@rolsonquadras any specific change you'd like to see?

Reply:
@llorllale looks like you have updated this with [new patch](https://github.com/hyperledger/aries-framework-go/compare/f4423945b781038c7655a19c0dbe820aee0c6944..151000517d2512bebf2d96f64b648a8e9a36f796#diff-195a635ad245ded9076330e31134bd80R4).